.molecule/default/files/polish/lib/workchain.py

```diff
@@ -185,7 +186,7 @@ def format_indent(level=0, width=INDENTATION_WIDTH):
     return ' ' * level * width
 
 
-def write_workchain(outlines, directory=None, filename=None):
+def write_workchain(outlines, directory=None):
     """
     Given a list of string formatted outlines, write the corresponding workchains to file
```

Comment:
and return the filename.

Since you anyhow switched to pathlib, perhaps it would be cleaner to return the path and then use `.name` in the cli

Reply:
e027fc995a2f58b74468e7d7608a6467633502f7